src/lib/humanBytes.test.js

```diff
@@ -0,0 +1,15 @@
+import humanBytes from "./humanBytes";
+
+describe("humanBytes", () => {
+  test.each([
+    [100, "100 B"],
+    [1400, "1.4 kB"],
+    [34567, "33.8 kB"],
```

Comment:
`kB` and `MB`? Shouldn't it be `KB`?

Reply:
Also, I guess we're fine with the kibibytes represented as kibibytes? That's what we agreed on back in the day and I'm fine with it, I'm just making sure.

Reply:
We'll probably make some changes to it (see [SKY-1121](https://linear.app/skynetlabs/issue/SKY-1121/use-mib-and-tib-prefix-in-dashboard-or-use-powers-of-10-insead-of-2-on)), but either way it's not the point of this PR :)